src/olympia/devhub/templates/devhub/includes/version_file.html

```diff
@@ -7,36 +7,5 @@
     <td>{{ file.size|filesizeformat(binary=True) }}</td>
     <td>{{ form.platform }}</td>
     <td>{{ choices[file.status] }}</td>
-    <td>
-      <span class="js-hidden">
-        <span class="delete">{{ form.DELETE }}</span>
-        {{ form.id }}
-      </span>
-      {# Don't let them delete an individual file any of the files have been reviewed. #}
-      {% if version and version.is_all_unreviewed: %}
-        <a class="remove" href="#">x</a>
```

Comment:
thing is, the `file-list` id is also used by `devhub/templates/devhub/addons/forms_shared/media.html` to edit previews...